README.md

```diff
@@ -5,7 +5,8 @@ A HTML5 fullscreen plugin for Leaflet.
 
 ``` js
 var map = new L.Map('map', {
-    fullscreenControl: true
+    fullscreenControl: true,
+    fullscreenPseudo: false // if true, fullscreen to page width and height
```

Comment:
Can you make this a sub-option of the `fullscreenControl` option? I.e., the usage would be:

```
new L.Map('map', {fullscreenControl: {pseudoFullscreen: true}});
```